### PR TITLE
Add article about NSSpain

### PR DESCRIPTION
**Pull request by nickskull**

### CODE REVIEW

File: source/_posts/2016-10-01-Insights-from-NSSpain.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+**Karaoke talks** were a unique and extremely entertaining concept which I have never seen at any other conference. Very similar to lightning talks except for one major difference - the speaker did not know nor see his slides beforehand and had to come up with the topic as they were being displayed. This resulted in very funny and non-sensical talks with a bit of geeky developer humour.
+
+And last but not least, it was in Spain - land of tapas, vino and mañana! ☀️🇪🇸
+
+## Summary
+
+What I like about conferences is that when get back, you just want to do things, try all the stuff you learned about, play with new frameworks and SDKs, you feel full of energy, very motivated and very productive. 
+
+Another great thing about NSSpain is that the city it's in, Logroño, is the capital of the Rioja region, a world famous wine region. And the day after NSSpain starts the 1-week long San Mateo wine festival. 🍷
```
Comment (mariusc) [inline source/_posts/2016-10-01-Insights-from-NSSpain.md]: It kind of feels like it's missing a paragraph at the end, it seems a bit unfinished. How about switching the two paragraphs and adding a bit more to them? Something like:

> Another great thing about NSSpain is that the city it's in, Logroño, is the capital of the Rioja region, a world famous wine region. And the day after NSSpain starts the 1-week long San Mateo wine festival 🍷, where NSSpain participants can join the festivities and enjoy the local products and atmosphere.
> 
> Probably the best part about conferences is that when you get back, you just want to do things, try all the stuff you learned, play with new frameworks and SDKs, you feel full of energy, very motivated and very productive. We feel that our team has gained a lot by taking part in NSSpain, and we'll try to keep the practice of attending conferences for the following years.
> 

But with this change, it looks like the first paragraph about San Mateo shouldn't belong under the "Summary" headline. 
What do you think?

**Comment by nickskull** [comment]
@mariusc Could you please make the changes if you have the time? :) Otherwise I'll get to it by the end of the week.

**Comment by mariusc** [comment]
@nickskull Should I fork and PR to your branch? Or will you give me commit access here? 😄

**Comment by nickskull** [comment]
@mariusc Just fork, not sure if I have enough rights to give you access :D

**Comment by mariusc** [comment]
@nickskull Opened PR #19. Please review :D. 
Opening a PR to a PR is so meta..

**Comment by nickskull** [comment]
Looks great! Thanks @mariusc :) Hopefully the article is live soon.

**Pull request merged**